### ISSUE TITLE
Fortran Committee

### ISSUE THREAD

**Issue by jacobwilliams** [j3-fortran/fortran_proposals#71]
I think it would be helpful if the committee produced some documentation on:

* What exactly is the Fortran committee?
* How does it currently work?
* How does it relate to the ISO committee? Are there two committees? Why is that?
* How does one join either or both of these committees?

**Comment by certik** [comment]
Fortran is an ISO standard. ISO has an international committee (https://wg5-fortran.org/), and national bodies. The US national body is (https://j3-fortran.org/).

Most of the work seems to happen in the J3 committee (the US national body).

To join the committee, simply join the J3 mailinglist, email Steve Lionel and start attending the meetings. Anybody can join, and you would be an observer. In order to be able to vote, you have to be part of some organization that is part of ISO (and pays the fees) and you become their representative.

I agree it would be nice to document this in the README of this repository, or elsewhere.

**Comment by sblionel** [comment]
https://software.intel.com/en-us/blogs/2011/09/22/the-real-doctors-of-fortran touches on this.

I am not the contact for J3, that would be Dan Nagle. The rules from INCITS (which manages US standards committees on behalf of ANSI) say that you may attend only twice as an "observer" - to attend more you have to join. If you want to join as your own member, as I do now, you have to pay INCITS each year, currently about $1300 for an individual. Or, you may get your employer or other organization to pay the fee. But many J3 members are "alternates" of primary members, and that is free. Generally we're always glad to have new members and will find someone to sponsor you as an alternate if needed. The caveat is that alternates don't get to vote in plenary if the primary member is present.

As for ISO, becoming a member of J3 automatically gets you an ISO membership (which has no fees.)

**Comment by sblionel** [comment]
I should also add that anyone can join the J3 mailing list - there's no relationship with membership. All we ask is that you have an interest in Fortran. Just go to https://mailman.j3-fortran.org/mailman/listinfo/j3 and fill in the join request. It will then be reviewed by list administrators, mainly to keep out spammers.

**Comment by marshallward** [comment]
Should we consider closing this issue and moving the information to the wiki?  It's not a language proposal, but it's invaluable information.

**Comment by certik** [comment]
@marshallward yes, let's move this into the Wiki, or even README. After that we can close this.
